addon/serializers/application.js

```diff
@@ -32,6 +32,9 @@ export default DS.JSONAPISerializer.extend({
             return Ember.String.camelize(key);
         }
     },
+    keyForRelationship(key) {
```

Comment:
This probably needs to borrow the pattern used in keyForAttribute (where serialize applies the inverse transformation of deserialize)